src/components/Map/Markers/index.tsx

```diff
@@ -1,15 +1,13 @@
 import React from "react";
 
 import ClusterMarkers from "./ClusterMarkers";
-import FacilityMarkers from "./FacilityMarkers";
 import PoiMarkers from "./PoiMarkers";
 import UserMarker from "./UserMarker";
 
 const Markers: React.FC = () => {
   return (
     <>
       <ClusterMarkers />
-      <FacilityMarkers />
```

Comment:
Please do not move any markers out of this file

Reply:
Even though i already contain it in the FacilityFilterFab? 
Won't duplicate cause problems?

Reply:
Using redux to store the selectedCategories